parser_library/src/workspaces/processor_file_impl.cpp

```diff
@@ -26,20 +28,26 @@ processor_file_impl::processor_file_impl(
     : file_impl(std::move(file_loc))
     , cancel_(cancel)
     , macro_cache_(file_mngr, *this)
-{}
+{
+    assert(fade_messages_);
```

Comment:
how could this not be true?

Reply:
Basically when malloc fails. This comes from embedded where we checked such things but I guess that on mainstream OSs this would be a sign of a bigger problem...